package.json

```diff
@@ -81,7 +81,7 @@
     "@mui/internal-scripts": "^1.0.18",
     "@mui/internal-test-utils": "1.0.11",
     "@mui/material": "6.0.1",
-    "@mui/monorepo": "github:mui/material-ui#v6.0.0-beta.4",
+    "@mui/monorepo": "github:michaldudak/material-ui#3b7d609357",
```

Comment:
This will be changed to mui/material-ui once https://github.com/mui/material-ui/pull/43574 is merged.